src/components/footer/__tests__/index.test.tsx

```diff
@@ -37,7 +38,19 @@ describe('Footer', () => {
   });
 
   describe('navigation buttons', () => {
-    it('opens about dialog', () => {
+    it('opens about dialog and display show more languages link when showMoreLanguagesLink is true', () => {
+      Config.showMoreLanguagesLink = true;
```

Comment:
Tests shouldn't be modifying the configuration directly. They should use `ConfigContext.Provider`. I think there's an example elsewhere?